components/notifier.rst

```diff
@@ -0,0 +1,174 @@
+.. index::
+   single: Notifier
+   single: Components; Notifier
+
+The Notifier Component
+====================
+
+    The Notifier component helps sending emails.
+
+If you're using the Symfony Framework, read the
+:doc:`Symfony Framework Notifier documentation </notifier>`.
+
+Installation
+------------
+
+.. code-block:: terminal
+
+    $ composer require symfony/notifier
+
+.. include:: /components/require_autoload.rst.inc
+
+Email
+-----
+
+The Notifier component has notify you when something goes wrong::
+
+    use Symfony\Bridge\Twig\Mime\NotificationEmail;
+    use Symfony\Component\Mailer\Mailer;
+    use Symfony\Component\Mailer\Transport\Smtp\EsmtpTransport;
+
+    $transport = new EsmtpTransport('localhost');
+    $mailer = new Mailer($transport);
+
+    $email = (new NotificationEmail())
+        ->form('fabien@symfony.com')
+        ->to('fabien@symfony.com')
+        ->exception($exception);
+
+    $mailer->send($email);
+
+The ``$email`` object is created via the :doc:`Mime component </components/mime>`.
+
+And configurable email template::
+
+    $email = (new NotificationEmail())
+        ->htmlEmail('email/system.html.twig')
+        ->textEmail('email/system.txt.twig');
+
+With template::
+
+    {% extends "@email/system.html.twig" %}
+    {% block style %}
+        {{ parent() }}
+        .container.body_alert {
+            border-top: 30px solid #ec5840;
+        }
+    {% endblock %}
+    {% block lines %}
+        This is an automated email for the MyApp application.
+        {{ parent() }}
+    {% endblock %}
+    {% block action %}
+        {{ parent() }}
+        <spacer size="16"></spacer>
+        <button class="secondary" href="https://myapp.com/">Go to MyApp</button>
+    {% endblock %}
+    {% block exception %}{% endblock %}
+    {% block footer_content %}
+        <p><small>&copy; MyApp</small></p>
+    {% endblock %}
+
+
+SMS
+---------
+
+Sending SMS Messages the easy way::
```

Comment:
SMS stands for Short Message Service
so Short Message Service Messages seems weird, what about

```suggestion
Sending SMS notifications the easy way::
```

?